DecipherNow/gm-fabric-dashboard
# feat: add German translations

Resolves #1502 

## Proposed Changes

  - Add German

Note: The numbers aren't getting properly internationalized. This will be fixed in a follow up issue. (#1525)

## How to Test

  - Start app
  - Select German
  - Confirm app loads as expected
